extensions/wikia/Chat2/js/views/views.js

```diff
@@ -426,7 +426,7 @@ var NodeChatUsers = Backbone.View.extend({
     		e.preventDefault();
     		var name = $(e.target).closest('.UserStatsMenu').find('.username').text();
     		if(!(name.length > 0)) {
```

Comment:
can we simplify this condition? i.e  `if (name.length === 0)` or even `if (name)` should also work